easybuild/easyconfigs/a/ASAP3/ASAP3-3.10.8-foss-2017b-Python-3.6.2.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = "PythonPackage"
+
+name = 'ASAP3'
+version = '3.10.8'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['a167e89c717cc75c1ff41bb370b9454563c29333e49b7383e668094af547d139']
+
+dependencies = [
+    ('Python', '3.6.2'),
```

Comment:
Maybe go with the latest `3.6.3` here? `Python` easyconfig is already available.